CSharp/BatchExplorer/Helpers/Common.cs

```diff
@@ -40,14 +40,26 @@ public static class Common
                 "A7",
                 "A8",
                 "A9",
+                "A10",
+                "A11",
                 "STANDARD_D1",
+                "STANDARD_D1_V2",
                 "STANDARD_D2",
+                "STANDARD_D2_V2",
                 "STANDARD_D3",
-                "STANDARD_D4",
```

Comment:
According to [the docs](https://docs.microsoft.com/en-us/azure/virtual-machines/virtual-machines-windows-sizes?toc=%2fazure%2fvirtual-machines%2fwindows%2ftoc.json#d-series), `STANDARD_D5` doesn't exist while `STANDARD_D5_V2` does.